### DISCUSSION TITLE
Server / Client

### DISCUSSION THREAD

**Discussion by mweel1** [vercel/turborepo#4505] [Help] [answered]
I have a server that is node.js and a client that is react.

I'm trying to figure out the best way to build the project, and then merge the build artifacts into my node js server so it can host the client code along with the server API.

Is this best done in my github actions with copy commands, or any other suggestions?

**Comment by anthonyshew** [comment, marked as answer]
It sounds like you'll want to get your build artifacts created and deploy them.

There aren't really enough details or specifics in this question to be able to help you much. Could you provide any information on where in particular you are getting stuck with your attempts and architecting?
